Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_dateperiod.py

```diff
@@ -806,8 +806,8 @@ def _parse_one_word_period(self, source: str, reference: datetime) -> DateTimeRe
             swift = self.config.get_swift_day_or_month(trimmed_source)
 
             if self.config.is_week_only(trimmed_source):
-                monday = DateUtils.this(reference, DayOfWeek.Monday) + datedelta(days=7 * swift)
-                result.timex = f'{year:04d}-W{monday.isocalendar()[1]:02d}'
+                thursday = DateUtils.this(reference, DayOfWeek.Thursday) + datedelta(days=7 * swift)
+                result.timex = f'{thursday.year:04d}-W{thursday.isocalendar()[1]:02d}'
```

Comment:
There is a wrapper method can be used to get ISO week number.

Reply:
Do you mean I should write "thursday.isocalendar()[0]" here?

Reply:
The variable 'monday' is only used to get the year number. I am trying to look almost the same in every language.

Reply:
I meant `thursday.isocalendar()[1]` can be replaced.

Reply:
As you wish.